defaults/test_default.yaml

```diff
@@ -242,3 +242,11 @@ validate_large_collections: false
 scylla_d_overrides_files: []
 
 run_commit_log_check_thread: true
+
+# teardown validators
+teardown_validators:
+  scrub:
+    enabled: false
```

Comment:
Maybe we should set it to "true" for a while to see what it catches and how stable the feature is.